# 存储地图服务
# 服务名: /lightning/save_map
# 启动slam节点后提供

string map_id        # 地图名称（唯一标识）
---
uint32 response     # 为0时成功， 1保存地图已经在运行 2图像渲染重置失败 3地图数据为空，无法保存，4扩图定位失败
